redis/asyncio/cluster.py

```diff
@@ -664,6 +670,11 @@ async def execute_command(self, *args: EncodableT, **kwargs: Any) -> Any:
                         raise RedisClusterException(
                             f"No targets were found to execute {args} command on"
                         )
+                    if (
+                        len(target_nodes) == 1
+                        and target_nodes[0] == self.get_default_node()
+                    ):
+                        is_default_node = True
```

Comment:
This check could be moved inside the `if self._initialize:` block.

Reply:
changed